src/test/java/org/fundacionjala/coding/wilson/SpinningWordsTest.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.coding.wilson;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class SpinningWordsTest {
+
+    /**
+     *
+     */
+
+    @Test
+    public void testSpin() {
+        assertEquals("emocleW to esuoh", new SpinningWords().spinWords("Welcome to house"));
+        assertEquals("emocleW", new SpinningWords().spinWords("Welcome"));
+        assertEquals("Hey wollef sroirraw", new SpinningWords().spinWords("Hey fellow warriors"));
+        assertEquals("This is a test", new SpinningWords().spinWords("This is a test"));
+        assertEquals("This is rehtona test", new SpinningWords().spinWords("This is another test"));
+        assertEquals("ylsuoireS this is the last one", new SpinningWords().spinWords("Seriously this is the last one"));
```

Comment:
You are instantiating all the time new SpinningWords()
please creare a variable for that.